assert/http_assertions_test.go

```diff
@@ -11,6 +12,12 @@ func httpOK(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 }
 
+func httpReadBody(w http.ResponseWriter, r *http.Request) {
+	_, _ = io.ReadAll(r.Body)
```

Comment:
No need for allocating memory to discard it.
```suggestion
	io.Copy(io.Discard, r.Body)
```